### ISSUE TITLE
[ML] Создать scripts/train_cnn.py

### ISSUE THREAD

**Issue by KalashnikovKV** [KalashnikovKV/Smart_storage#31]
## Задача
Создать `scripts/train_cnn.py` — fine-tune MobileNetV3 на ROI-изображениях объектов.

## Описание
```bash
uv run python scripts/train_cnn.py \
    --data  dataset/labeled/ \
    --epochs 30 \
    --output models/cnn_classifier.pth
```

Шаги:
1. Загрузить `dataset/labeled/{class}/` через `ImageFolder`
2. Аугментации: RandomHorizontalFlip, ColorJitter, RandomRotation (через albumentations)
3. Fine-tune MobileNetV3-Small (pretrained=True, заменить classifier head)
4. Сохранить веса + `class_names.json`

## Критерий готовности
- [ ] Fine-tune работает на CPU и GPU
- [ ] Выводит train/val accuracy по эпохам
- [ ] Сохранённые веса загружаются в `CNNClassifier` (#11)

## Зависит от
#9, #11, #14

**Comment by KalashnikovKV** [comment]
Закрываем: заменяется scripts/train_yolo.py. Fine-tune CNN не нужен — обучаем YOLO-Seg модель.
